tests/test_ops.py

```diff
@@ -1734,7 +1743,12 @@ def test_mathtransform():
         # {"opName": "reduce_mean", "outName": "ragged/reduce_mean/2d_a1", "varShapes":[[]], "varTypes":["float32"], "varInit":["ragged2d"], "axis":1, "keepdims":False},
         # {"opName": "sqrt", "outName": "ragged/sqrt/2d", "varShapes":[[]], "varTypes":["float32"], "varInit":["ragged2d"]},
 
-        {"opName": "strings_split", "outName": "ragged/sqrt/2d", "varShapes":[[]], "varTypes":["string"], "varInit":["string2"], "split":" "},
+        # {"opName": "strings_split", "outName": "ragged/sqrt/2d", "varShapes":[[]], "varTypes":["string"], "varInit":["string2"], "split":" "},
+        # {"opName": "bitcast", "outName": "bitcast", "varShapes":[[1,10]], "varTypes":["float32"], "varInit":["uniform", "uniform"], "dtype":tf.float32},
+        # {"opName": "bitwise_or", "outName": "bitwise_or", "varShapes":[[1,10], [1,10]], "varTypes":["int64", "int64"], "varInit":["uniform_int10", "uniform_int10"]},
+        # {"opName": "bitwise_xor", "outName": "bitwise_xor", "varShapes":[[1,10], [1,10]], "varTypes":["int64", "int64"], "varInit":["uniform_int10", "uniform_int10"]},
+        # {"opName": "is_non_decreasing", "outName": "is_non_decreasing", "varShapes":[[1,10]], "varTypes":["float32"], "varInit":["uniform"]},
+        #  {"opName": "is_strictly_increasing", "outName": "is_strictly_increasing", "varShapes":[[1,10]], "varTypes":["float32"], "varInit":["uniform"]},
```

Comment:
Again, improved coverage. Rank 0, 1, 2 + different datatypes (int8, int32, bfloat16, float32, float64)
Also be careful with the initialization here. Uniform initialization on [1,10] array will almost always be non-decreasing (not increasing) in practice -> we're never testing one side of the condition.

Reply:
Fixed